homeassistant/components/heos/services.yaml

```diff
@@ -1,3 +1,17 @@
+group_volume_set:
```

Comment:
`set_group_volume` sounds better

Reply:
I agree, however 'volume_set' is the name of the existing service, so was trying to keep it parallel with the naming convention we have. Is it acceptable to do something different here?

Reply:
Hmm, maybe it makes sense, but I don't have strong feelings for it